__test__/integrations/contact-loaders/ngpvan/ngpvan.test.js

```diff
@@ -0,0 +1,1066 @@
+
+      expect(JSON.parse(savedListsResponse.data).items).toEqual(listItems);
+      expect(savedListsResponse.expiresSeconds).toEqual(30);
+      getSavedListsNock.done();
+    });
+
+    describe("when there is an error retrieving the list", () => {
+      it("returns what we expect", async () => {
+        const getSavedListsNock = nock(`${fakeNgpVanBaseApiUrl}:443`, {
+          encodedQueryParams: true,
+          reqheaders: {
+            authorization: "Basic c3Bva2U6dG9wc2VjcmV0fDA="
+          }
+        })
+          .get(
+            `/v4/savedLists?$top=&maxPeopleCount=${process.env.NGP_VAN_MAXIMUM_LIST_SIZE}`
+          )
+          .reply(404);
+
+        const savedListsResponse = await getClientChoiceData();
+
+        expect(JSON.parse(savedListsResponse.data)).toEqual({
+          error: expect.stringMatching(
+            /Error retrieving saved list metadata from VAN Error: Request id .+ failed; received status 404/
+          )
+        });
+        getSavedListsNock.done();
+      });
+    });
+  });
+
+  describe("@processContactLoad", () => {
+    let job;
+    let payload;
+    let maxContacts;
+    let csvReply;
+    let oldNgpVanWebhookUrl;
+    let oldNgpVanExportJobTypeId;
+    let oldNgpVanAppName;
+    let oldNgpVanApiKey;
+    let makeSuccessfulExportJobPostNock;
+    let makeSuccessfulGetCsvNock;
+    let webhookUrl;
+    let oldNgpVanApiBaseUrl;
+    let organization;
+
+    beforeEach(async () => {
+      oldNgpVanWebhookUrl = process.env.NGP_VAN_WEBHOOK_BASE_URL;
+      oldNgpVanExportJobTypeId = process.env.NGP_VAN_EXPORT_JOB_TYPE_ID;
+      oldNgpVanAppName = process.env.NGP_VAN_APP_NAME;
+      oldNgpVanApiKey = process.env.NGP_VAN_API_KEY;
+      oldNgpVanApiBaseUrl = process.env.NGP_VAN_API_BASE_URL;
+      process.env.NGP_VAN_WEBHOOK_BASE_URL = "https://www.example.com";
+      process.env.NGP_VAN_EXPORT_JOB_TYPE_ID = 7;
+      process.env.NGP_VAN_APP_NAME = "spoke";
+      process.env.NGP_VAN_API_KEY = "topsecret";
+      process.env.NGP_VAN_API_BASE_URL = fakeNgpVanBaseApiUrl;
+    });
+
+    beforeEach(async () => {
+      payload = {
+        savedListId: 682951,
+        savedListName: "democrats on 103rd Street"
+      };
+
+      job = {
+        id: 7,
+        campaign_id: 1,
+        payload: JSON.stringify(payload)
+      };
+
+      organization = {
+        id: 77,
+        name: "What good shall I do today?"
+      };
+    });
+
+    beforeEach(async () => {
+      csvReply = `CanvassFileRequestID,VanID,Address,FirstName,LastName,StreetAddress,City,State,ZipOrPostal,County,Employer,Occupation,Email,HomePhone,IsHomePhoneACellExchange,CellPhone,WorkPhone,IsWorkPhoneACellExchange,Phone,OptInPhone,OptInStatus,OptInPhoneType,CongressionalDistrict,StateHouse,StateSenate,Party,PollingLocation,PollingAddress,PollingCity
+1286,6144436,"1749 Kori Ter, Ranbelsi, SC 02569",Jean,Leclerc,1749 Kori Ter,Ranbelsi,SC,,Suffolk,,,,,,,,,,,,,001,004,002,R,,,
```

Comment:
I know these are fake, but in the future, please use more obviously fake data so it's easier to vet whether personal data has somehow gotten added to the repo.  Data like this should more appropriately go in `__test__/test_data/`